elastic/kibana
# Add new telemetry data from event-log index.

Resolves #138996

This PR is a follow-on of https://github.com/elastic/kibana/pull/139901.
And covers the rest of the requested telemetry data(numbers 7-9 in the issue) by aggregating data from event-log index  

To Verify:

- Create a couple of rules with always triggering conditions (e.g. index-threshold that checks if the doc count is above 0)
- add at least 1 action.

Change alerting telemetry task run interval to something very short time (e.g. 1 min) 
https://github.com/elastic/kibana/blob/01ecbd48f3657327cb5375b54195cd07bfc4027f/x-pack/plugins/alerting/server/usage/task.ts#L197

Do the same for the actions telemetry task too
https://github.com/elastic/kibana/blob/01ecbd48f3657327cb5375b54195cd07bfc4027f/x-pack/plugins/actions/server/usage/task.ts#L143

Check the below data on `https://localhost:5601/api/stats?extended=true&legacy=true`
`count_rules_by_execution_status_per_day`, 
`count_connector_types_by_action_run_outcome_per_day`, 
`avg_actions_run_duration_by_connector_type_per_day`,

Edit: Realised that `avg_actions_run_duration_by_connector_type_per_day` is already in telemetry with the name `avg_execution_time_by_type_per_day`, therefore removed that field and its tests.
